File: fixtures/scale/php/large/src/Store/StorePaths.php
<?php

namespace CipherLab\Store;

class StorePaths
{
    public static function storeDir(): string
    {
        return getcwd() . '/.cipherlab';
    }

    public static function resultsFile(): string
    {
        return self::storeDir() . '/results.jsonl';
    }

    public static function ensureDir(): void
    {
        if (!is_dir(self::storeDir())) {
            mkdir(self::storeDir(), 0777, true);
        }
    }
}

File: fixtures/scale/csharp/large/Store/StorePaths.cs
using System;
using System.IO;

namespace CipherLab.Store
{
    public static class StorePaths
    {
        public static string StoreDir()
        {
            return Path.Combine(Directory.GetCurrentDirectory(), ".cipherlab");
        }

        public static string ResultsFile()
        {
            return Path.Combine(StoreDir(), "results.jsonl");
        }

        public static void EnsureDir()
        {
            Directory.CreateDirectory(StoreDir());
        }
    }
}
